File: 1027c/Hw34.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
int main() {
	int count,remember;
	char input[20];
	char name[5][200] = { "John Smith","Jackie Jackson","Chris Jones","Amanda Cullen","Jeremy Goodwin" };
	printf("There are 5 employees\n");
	for (count = 0; count < 5; count++) {
		printf("%s\n",name[count]);
	}
	printf("Enter the name to remove : ");
	fgets(input,sizeof(input),stdin);
	input[strlen(input) - 1] = '\0';
	remember = -1;
	for (count = 0; count < 5; count++) {
		if (!strcmp(input, name[count])) {
			remember = count;
			break;
		}
	}
	if (remember != -1) {
		printf("There are 4 employess.\n");
		for (count = 0; count < 5; count++) {
			if (count != remember) {
				printf("%s\n",name[count]);
			}
		}
	}
	else {
		printf("We don't know about %s",input);
	}
	system("pause");
}

File: 1026java/Hw34.java
import java.util.ArrayList;
import java.util.Scanner;
public class Hw34 {
    public static void main(String[] args) {
        Scanner Scan = new Scanner(System.in);
        ArrayList Employee = new ArrayList();
        Employee.add("John Smith");
        Employee.add("Jackie Jackson");
        Employee.add("Chris Jones");
        Employee.add("Amanda Cullen");
        Employee.add("Jeremy Goodwin");
        int sizeofArray = Employee.size();
        int count = 0;
        System.out.print("There are "+sizeofArray+" employees\n");
        for(count = 0; count < sizeofArray;count++){
            System.out.print(Employee.get(count)+"\n");
        }
        System.out.print("Enter the employee name to remove ");
        String employeesName = Scan.nextLine();
        if(Employee.contains(employeesName)){
            Employee.remove(employeesName);
            sizeofArray = Employee.size();
            System.out.print("There are "+sizeofArray+" employees\n");
            for(count = 0; count < sizeofArray;count++){
                System.out.print(Employee.get(count)+"\n");
            }
        }
        else{
            System.out.print("You entered wrong name");
        }

    }
}
